packages/zowe-explorer/src/SpoolProvider.ts

```diff
@@ -70,6 +52,22 @@ export const toUniqueJobFileUri =
         });
     };
 
+/**
+ * Helper function called by {@link toUniqueJobFileUri}
+ * Encode the information needed to get the Spool content.
+ *
+ * @param profileName The name of the Zowe profile to use to get the Spool Content
+ * @param spool The IJobFile to get the spool content for.
+ */
+export function encodeJobFile(profileName: string, spool: zowe.IJobFile): vscode.Uri {
```

Comment:
Curious why we are keeping this function around, but moving it to a different spot in the file? 🤔

Reply:
for some reason this was marked deprecated but another function in same file is calling it.  for now I just moved it below the function that called it and updated the comments while un-deprecating it